test/Microsoft.Extensions.DependencyModel.Tests/PackageResolverTest.cs

```diff
@@ -24,8 +24,8 @@ public void ShouldUseEnvironmentVariableToGetDefaultLocation()
                 .AddVariable("NUGET_PACKAGES", PackagesPath)
                 .Build();
 
-            var result = PackageCompilationAssemblyResolver.GetDefaultPackageDirectory(Platform.Unknown, environment);
-            result.Should().Be(PackagesPath);
+            var result = PackageCompilationAssemblyResolver.GetDefaultProbeDirectories(Platform.Unknown, environment);
```

Comment:
I see now what you were asking about last night w.r.t. testing DependencyModel with the latest Microsoft.NETCore.App.

Would it be possible to get some sort of end-to-end test here?  Basically a 2.0 app that references DependencyModel, call it with `--additionalprobingpath`, and ensure that the PackageCompilationAssemblyResolver can find assemblies in that probing path?